<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>game_over</class>
 <widget class="QWidget" name="game_over">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="header_lable">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>451</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Time's up!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="dated_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>You dated x spaceships!</string>
   </property>
  </widget>
  <widget class="QLabel" name="rejected_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>491</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>You got rejected by x spaceships</string>
   </property>
  </widget>
  <widget class="QLabel" name="points_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>441</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>30</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Your points :  xx</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>590</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>game_over</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>618</y>
    </hint>
    <hint type="destinationlabel">
     <x>525</x>
     <y>554</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
